sirepo/auth/email.py

```diff
@@ -59,11 +59,11 @@ def api_authEmailAuthorized(self, simulation_type, token):
         """
         if http_request.is_spider():
             sirepo.util.raise_forbidden('robots not allowed')
-        req = http_request.parse_params(type=simulation_type)
+        req = self.parse_params(type=simulation_type)
```

Comment:
remove the import of http_request here and any other place it is not needed.

Reply:
this file still uses http_request.is_spider() on line 60, did you want me to remove that too?

Reply:
no. let's do this incrementally. only modules which no longer rely on http_request should have their imports removed.